src/Microsoft.OpenApi.Readers/V31/OpenApiSecurityRequirementDeserializer.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers.ParseNodes;
+
+namespace Microsoft.OpenApi.Readers.V31
+{
+    /// <summary>
+    /// Class containing logic to deserialize Open API V31 document into
+    /// runtime Open API object model.
+    /// </summary>
+    internal static partial class OpenApiV31Deserializer
+    {
+        public static OpenApiSecurityRequirement LoadSecurityRequirement(ParseNode node)
+        {
+            var mapNode = node.CheckMapNode("security");
+            string description = null;
+            string summary = null;
+
+            var securityRequirement = new OpenApiSecurityRequirement();
+
+            foreach (var property in mapNode)
+            {
+                if (property.Name.Equals("description") || property.Name.Equals("summary"))
```

Comment:
I don't believe this is correct.  SecurityRequirement references are not technically Reference Objects in OAS so you can't have description and summaries along side them.

Reply:
Same for Tags.  We cannot override descriptions for these references.